app/assets/stylesheets/admin.scss

```diff
@@ -125,3 +125,21 @@ label {
   flex-direction: column;
   gap: var(--su-4);
 }
+
+.admin-config-checkmark {
```

Comment:
I added these classes to reproduce the green `Enabled` checkmark, and to render the `not-allowed` cursor when the user hovers over the disabled `Invite Only Mode` checkbox.